app/controllers/admin/users/gdpr_delete_requests_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Admin
+  module Users
+    class GdprDeleteRequestsController < Admin::ApplicationController
+      layout "admin"
+
+      def index
+        @gdpr_delete_requests = ::Users::GdprDeleteRequest.order("created_at desc").page(params[:page]).per(50)
+      end
+
+      def destroy
+        @gdpr_delete_request = ::Users::GdprDeleteRequest.find(params[:id])
```

Comment:
Since the deletion is the confirmation by the admin that they removed the data, I think we should log this. Something like:

```ruby
AUDIT_LOG_CATEGORY = "admin.gdpr_delete.confirm".freeze

def destroy
  ...
  AuditLog.create(
    user: current_user, 
    category: AUDIT_LOG_CATEGORY,
    roles: current_user.roles_name,
    ...
)
```